lib/active_job/queue_adapters/solid_queue_ext.rb

```diff
@@ -198,6 +198,7 @@ def order_executions(executions)
           case
             # Follow polling order for scheduled executions, the rest by job_id, desc or asc
           when solid_queue_status.scheduled? then executions.ordered
+          when solid_queue_status.failed? then executions.order(created_at: :desc)
```

Comment:
We don't have a good index for this, so this won't work in general in the case you have many failed jobs 🤔 We'll need a good index first.

Reply:
Thanks, indeed !

happy to help on this point, but I'm not sure of the process, especially to provide migrations to end users through an update, does it necessarily have to be done through a generator like the install migrations ?